### ISSUE TITLE
java.io.FileNotFoundException : All image file can't be found after upgrading the version to 2.0.5

### ISSUE THREAD

**Issue by QiuhengYi** [docToolchain/docToolchain#845]
I upgrade the version of docToolChain from 1.3.1 to 2.05, then the problem happens:

    Where:
    Script '/home/dtcuser/docToolchain/scripts/publishToConfluence.gradle' line: 26

    What went wrong:
    Execution failed for task ':publishToConfluence'.
    > java.io.FileNotFoundException: /project/documentation/documentation/build/html5/images/100_cloud-infrastructure/101_service_offering_and_onboarding/101_subscription.png (No such file or directory)

But I didn't change the directory of the image path.
The only thing I did was upgrading the version to 2.0.5.

**Comment by jmini** [comment]
The path: `/project/documentation/documentation` looks wired.

Have you a small example public repository that shows the problem?

**Comment by rdmueller** [comment]
yes, the path looks wired. We have some breaking changes from 1.3.x to 2.x and I think you discovered one. I guess the config has changed a bit in regards to the source folder.
Can you please run `./dtcw` with the `--info` option? This will give us some insight on how the path is configured.

**Comment by ehmkah** [comment]
Are there any news on this? It looks somehow similar to https://github.com/docToolchain/docToolchain/issues/883 it looks like this behavior is also in doctoolchain 2.1.0

What I observe is that the images which were generated from external graphics e.g. graphiz with 
```
   [graphviz, format=png]
   ---
   strict digraph G {}
   ---
```

were generated and only the "real" images were missing.

My current workaround for this is:
- doctoolchain generateHTML
- copy manually images from images folder to folder of where generateHTML stored its generated html files which is `build/html5/images 
- doctoolchain publishConfluence

**Comment by PacoVK** [comment]
Can someone please check if the issue persists with 3.1.2? With the `failOnMissingImages = true` property the images should be handled properly without having to copy them over
